src/pages/Profile/Profile.css

```diff
@@ -13,6 +14,52 @@
          This will help you when you get started.
     */
 
+.pic{
+    background-color: rgb(191, 191, 191);
+    
+    width: 300px;
+    height: 300px;
+    border-radius: 50%;
+    margin-left: 200px;
```

Comment:
Instead of having a margin on this specific element, I would suggest adding padding on its parent element (`profilePage`) so that the right also has the same padding for the symmetry.